### DISCUSSION TITLE
Kio-Fuse use in LXQt

### DISCUSSION THREAD

**Discussion by maverick74** [lxqt/lxqt#1963] [Q&A] [answered]
This is not exactly an issue but more a feature request or an attention call to bring discussion about it (if it worth's it... naturally).

KDE's Plasma shipped Kio-Fuse. It allows one to access remote files and work with them easily like they're on local disk.

Is it something that can be integrated into LXQt? Is it worth the effort?

**Comment by tsujan** [comment, marked as answer]
We use GIO+GVFS, not KIO.

Actually, KIO is behind GIO in file management.

**Reply by maverick74** [reply]
GIO?!... never heard of it. I suppose it stands for "Gnome IO".
I'm going to investigate it, as i'm having a few network problems on suse (thank you for the tip ;) )

**Reply by tsujan** [reply]
GIO: https://en.wikipedia.org/wiki/GIO_(software)
GVFS: https://en.wikipedia.org/wiki/GVfs

Yes, developed by Gnome devs but not related to GTK.
